modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSQuickActions.testcase

```diff
@@ -65,6 +65,18 @@ definition {
 		}
 	}
 
+	@description = "LPS-155463. Assert that Quick actions icons list should be limited to three actions."
+	@priority = "4"
+	test CanDisplayUpToThreeIcons {
+		task ("Then quick action menu only has 3 actions icons available") {
+			FDSQuickActions.viewQuickActionMenuPresent(title = "Sample1");
+
+			takeScreenshot();
+
+			SikuliAssertElementPresent(locator1 = "FrontendDataSet#PREVIEW_QUICK_ACTION_ICON");
```

Comment:
Should we use Sikuli? Or attempt to use Ocular to assert only 3 quick actions? @john-co

Reply:
This is really the only way I can think of to solve the **ONLY 3** problem.

Reply:
I want to avoid using sikuli. Another alternative is to check that the 3rd quick action is present and that the 4th quick action is not present using path locator indexes ie: *[3], *[4].

Reply:
For ocular...I was thinking about this yesterday and due to the pixel flakiness between CI and different local machine specs, I'd probably reserve ocular for asserting really janky bugs. ie: the whole page formatting is broken, or some component is completely missing. Icons may be too small and noisy for ocular...even if we go down to 20% sensitivity I'm just not confident for this particular usecase.